src/pages/collators/CollatorRewards.tsx

```diff
@@ -0,0 +1,160 @@
+      if (isDelegator) {
+        setUserStaking({
+          candidateId: candidate.id,
+          amount: isDelegator.amount,
+        });
+      }
+    });
+  }, [candidates, userAccountAddress, setUserStaking]);
+
+  useEffect(() => {
+    const fetchAvailableBalance = async () => {
+      if (!api || !walletAccount) {
+        return '0';
+      }
+      const { data: balance } = await api.query.system.account(walletAccount?.address);
+      return balance.free.sub(balance.miscFrozen).toString();
+    };
+
+    fetchAvailableBalance().then((balance) => setUserAvailableBalance(balance));
+  }, [api, walletAccount]);
+
+  const updateRewardsExtrinsic = useMemo(() => {
+    if (!api) {
+      return undefined;
+    }
+
+    return createUpdateDelegatorRewardsExtrinsic();
+  }, [api, createUpdateDelegatorRewardsExtrinsic]);
+
+  const submitUpdateExtrinsic = useCallback(() => {
+    if (!updateRewardsExtrinsic || !api || !walletAccount) {
+      return;
+    }
+    setSubmissionPending(true);
+    updateRewardsExtrinsic
+      .signAndSend(walletAccount.address, { signer: walletAccount.signer as any }, (result) => {
+        const { status, events } = result;
+        const errors = getErrors(events, api);
+        if (status.isInBlock) {
+          if (errors.length > 0) {
+            const errorMessage = `Transaction failed with errors: ${errors.join('\n')}`;
+            console.error(errorMessage);
+            toast(errorMessage, { type: 'error' });
+          }
+        } else if (status.isFinalized) {
+          setSubmissionPending(false);
+          if (errors.length === 0) {
+            toast('Delegator rewards updated', { type: 'success' });
+          }
+        }
+      })
+      .catch((error) => {
+        toast('Transaction submission failed: ' + error.toString(), {
+          type: 'error',
+        });
+        setSubmissionPending(false);
+      });
+  }, [api, setSubmissionPending, updateRewardsExtrinsic, walletAccount]);
+
+  return (
+    <>
+      <div className="flex mb-8 justify-between">
+        <div className="card gap-0 rounded-lg bg-base-200 w-1/2 mr-4 collators-box">
+          <div className="card-body">
+            <h2 className="card-title">Collators</h2>
+            <div className="flex flex-row">
+              <div className="flex-initial pr-5">
+                <StakedIcon />
+              </div>
+              <div className="flex-auto">
+                <h3>{nativeToFormat(userStaking?.amount || '0.00', tokenSymbol)}</h3>
+                <p>My Staking</p>
+              </div>
+              <div className="flex-auto">
+                <h3>{nativeToFormat(userAvailableBalance, tokenSymbol)}</h3>
+                <p>Free balance</p>
+              </div>
+              <div className="flex flex-auto place-content-end">
+                <button className="btn btn-secondary w-full" disabled>
+                  0 {tokenSymbol} Unboarding
```

Comment:
Should this really be 'Unboarding' or 'Unbonding'?

Reply:
Uhm, so the texts in the designs read 'Unboarding'. But maybe it was a misheard ? @prayagd ?

Reply:
I will mark it as solved so we can merge.